packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -350,3 +355,50 @@ function EmptyTest() {
     </div>
   );
 }
+
+
+function DragExample(props?) {
+  let getItems = (keys) => [...keys].map(key => {
+    let item = items.find(item => item.key === key);
+    return {
+      'text/plain': item.textValue
+    };
+  });
+
+  let itemAllowsDragging = (key) => {
+    let item = items.find(item => item.key === key);
+    return item.isDraggable;
+  };
+
+  return (
+    <>
+      <input />
+      <Droppable />
+      <ListView width="300px" selectionMode="multiple" items={items} getItems={getItems} itemAllowsDragging={itemAllowsDragging} {...props}>
+        {(item: any) => (
+          <Item key={item.key} textValue={item.textValue}>
+            <Content>
+              <Flex alignItems="center" gap="10px">
+                <View flexGrow={1}>Item {item.key}</View>
+                <ActionButton><Add /></ActionButton>
+                <MenuTrigger>
+                  <ActionButton><MoreSmall /></ActionButton>
+                  <Menu>
+                    <Item>
+                      <Edit />
+                      <Text>Edit</Text>
+                    </Item>
+                    <Item>
+                      <Delete />
+                      <Text>Delete</Text>
+                    </Item>
+                  </Menu>
+                </MenuTrigger>
```

Comment:
Found a problematic bug specifically w/ MenuTriggers in draggable rows. https://github.com/adobe/react-spectrum/blob/9ba765f71bdde97d1545d91f76326e61bd35ae30/packages/%40react-aria/interactions/src/usePress.ts#L763-L766 
The above will stop `onPointerDown` and `onMouseDown` from calling `e.preventDefault` meaning the browser will handle focusing the trigger button. This result in the following focus flow when clicking the menu trigger button: 
![image](https://user-images.githubusercontent.com/9661127/144336258-f882716c-c3da-43fb-8ae5-6fd930ac52f9.png)
The above flow is the button being focused -> menu opens and the first item is focused -> button is refocused by the browser (perhaps [noted here?](https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/interactions/src/usePress.ts#L355-L360)) and the menu closes. 

Problem is we actually want this preventDefault not to happen so that dragging works. Perhaps MenuTrigger should  open on press/press start if we detect that the trigger is in a draggable row/element? Kinda similar to the ListViewItem `shouldSelectOnPressUp` problem mentioned previously